ops/ansible/playbooks-ccs/vars/test/group_vars/all/env_specific.yml

```diff
@@ -51,6 +51,9 @@ data_server_ssl_client_certificates:
   - alias: client_performance_tester
     certificate: "{{ lookup('file', 'files/client_data_server_performance_tester_certificate.pem') }}"
 
+# Enables the CCW RIF job when true; disables when false
+data_pipeline_ccw_rif_job_enabled: false
```

Comment:
can you put a comment in here indicating the temporary nature of this for synthea load?